src/main/resources/application-local.yml

```diff
@@ -1260,5 +1260,31 @@ transfer:
           variableName: errorInformation
         - field: clientCorrelationId
           variableName: clientCorrelationId
+    - name: payment_notification
+      direction: OUTGOING
+      type: TRANSFER
+      transformers:
+        - field: amount
```

Comment:
please check the indentation with respect to other lines in this file

Reply:
okay

Reply:
yes it is correct